02week/blog/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<!-- saved from url=(0052)file:///C:/Users/DW/Desktop/htmlcheatsheet/blog.html -->
+<html <head=""><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252"><title>My Blog</title>
+    <meta name="description" content="awesome blog by will ragan">
+    <meta name="keywords" content="web design blog, web dev blog, Will Ragan">
+  </head>
+  <body><header id="Main-header">
+    <h1>My Blog</h1>
+  </header>
+
+  <a href="file:///C:/Users/DW/Desktop/htmlcheatsheet/index.html">Go to index</a>
```

Comment:
Let's check the path on this again; everything is relative so you don't need to include everything from your CL/Users path.. if you want to include another file, you do need to bring it into your project first